Add other methods of saving drafts (#47)
ezsystems/user-documentation: docs/publishing/publishing.md

Choosing **Save** does not immediately publish the Content item.
Instead, it will save the new version of the content as a **Draft**.
You can then keep on editing the same content and save it again when needed.

> You can then leave the editor by using the **X** icon in the upper left corner, it will also save a working copy as a draft.

Reviewer: ```suggestion
You can then leave the editor by using the **X** icon in the upper left corner, it will also save the working copy as a draft.
```

!!! caution

>     Only by selecting **Delete draft** the whole draft and all the work you have done on it will be removed. A discarded draft is removed instantly, not placed in the Trash, so it can't be restored.

Reviewer: ```suggestion
    Only selecting **Delete draft** removes the whole draft and all the work you have done on it.
    A discarded draft is removed instantly, not placed in the Trash, so it can't be restored.
```

>     The **X** icon in the upper left corner and browser’s **back button** will save the current Content item as a draft.

Reviewer: ```suggestion
    The **X** icon in the upper left corner and the browser's **back button** will save the current Content item as a draft.
```
